### ISSUE TITLE
Stage1: Utility Implementation

### ISSUE THREAD

**Issue by addy1997** [addy1997/mipnerf-pytorch#4]
Understand and implement the learning rate schedule.
